services.py

```diff
@@ -70,3 +72,28 @@ def update_or_create_criterion(data, criterion=None):
         criterion.deactivate()
 
     return criterion.save()
+
+
+def state_information(state):
```

Comment:
Is there a standard way to name a method like this? (It returns the results of a query.)